### PR TITLE
Fix volcano sounds

### LINKED ISSUE #6650: Adventure map, wrong ambient sound for lava objects
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Windows

### Describe the bug

Currently two lava objects have wrong sounds.
Volcano 2
Volcano 1
05. wav composition should be played while hero standing near these two objects.

### Save file

[Lava.zip](https://github.com/ihhub/fheroes2/files/10743138/Lava.zip)

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by oleg-derevenetz**
fix #6650

### CODE REVIEW

**Review by Branikolog — APPROVED** [review]

File: src/fheroes2/agg/m82.cpp [hunk 1]
```diff
@@ -214,6 +224,19 @@ M82::SoundType M82::getAdventureMapObjectSound( const MP2::MapObjectType objectT
...
+        case MP2::OBJ_ICN_TYPE_UNKNOWN:
+            return UNKNOWN;
+        // Small volcanoes
+        case MP2::OBJ_ICN_TYPE_OBJNLAVA:
+            return LOOP0005;
+        default:
+            break;
+        }
+        // Other volcanoes
+        return LOOP0027;
```
Comment (ihhub) [inline src/fheroes2/agg/m82.cpp:239]: I assume that we are not fixing the problem with multiple tiles having the same object ICN type and object ID as written in the TODO comments above but only separating sounds by object ICN types?

**Review by ihhub — COMMENTED** [review]
Hi @oleg-derevenetz , I left just one question here. Could you please take a look?

File: src/fheroes2/agg/m82.cpp (hunk 1 above)
Comment (oleg-derevenetz) [reply src/fheroes2/agg/m82.cpp:239]: @ihhub No, that's why I left the corresponding TODO comment in place. Only proper sound depending on the volcano type is implemented.

Comment (ihhub) [reply src/fheroes2/agg/m82.cpp:239]: Thank you for the confirmation!

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@oleg-derevenetz , thank you so much for this fix!

**Pull request merged**